app/Models/Enums/AcarsSource.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Models\Enums;
+
+use App\Contracts\Enum;
+
+/**
+ * Notice for 3rd party developers.
+ *
+ * This file defines the master integer value for sourcing of telemetry, logs, etc.
+ *
+ * If you are a developer and want to add your software to this file, please make a pull request with your change to the phpVMS repository,
+ * even if you don't intend to release your software publicly and will only be used by your community/VA. This way, integer values will not
+ * conflict across systems.
+ *
+ * When adding your client, please follow the comment formatting so that developers know, at a glance, who maintains the specific software.
+ *
+ * Company Name or Real Name (github)
+ */
+class AcarsSource extends Enum
+{
+    /* Desktop Based Clients */
+    public const VMSACARS = 0; // Nabeel S. (nabeelio)
+    public const SMARTCARS = 1; // Invernyx d.b.a. TFDi Design (invernyx)
+
+    /* Online Networks */
+    public const VATSIM = 50; // N/A
+    public const IVAO = 51; // N/A
```

Comment:
These two definitions I'm currently touchy on. Since VATSIM and IVAO are directly not sending, nor does phpVMS itself receive, data, I think we remove the generic enums. Since my module that'll do this is CHNetTRAK, I could do `NET_TRAK_VATSIM` and `NET_TRAK_IVAO`.

Thoughts?

Reply:
File deleted